<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogELFType</class>
 <widget class="QDialog" name="DialogELFType">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>104</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>291</width>
    <height>104</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>291</width>
    <height>104</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Type</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBoxX" name="checkBoxReadOnly">
       <property name="text">
        <string>Read only</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEditX" name="lineEditValue">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBoxX" name="comboBoxValue">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonOK">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonCancel">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonApply">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLineEditX</class>
   <extends>QLineEdit</extends>
   <header>qlineeditx.h</header>
  </customwidget>
  <customwidget>
   <class>QPushButtonX</class>
   <extends>QPushButton</extends>
   <header>qpushbuttonx.h</header>
  </customwidget>
  <customwidget>
   <class>QCheckBoxX</class>
   <extends>QCheckBox</extends>
   <header>qcheckboxx.h</header>
  </customwidget>
  <customwidget>
   <class>QComboBoxX</class>
   <extends>QComboBox</extends>
   <header>qcomboboxx.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
